packages/zeronetworks/data_stream/audit/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,755 @@
+---
+description: Pipeline for processing Zero Networks Audit logs
+processors:
+- rename:
+    field: message
+    target_field: event.original
+- json:
+    field: event.original
+    target_field: json
+- date:
+    field: json.isoTimestamp
+    formats:
+    - date_optional_time
+    target_field: '@timestamp'
+- convert:
+    field: json.auditType
+    type: string
+    ignore_missing: true
+- rename:
+    field: json.auditType
+    target_field: event.code
+    ignore_missing: true
+- rename:
+    field: json.performedBy.id
+    target_field: user.id
+    ignore_missing: true
+- rename:
+    field: json.performedBy.name
+    target_field: user.full_name
+    ignore_missing: true
+- rename:
+    field: json.enforcementSource
+    target_field: zeronetworks.audit.enforcementSource
+    ignore_missing: true
+    if: 'ctx.zeronetworks?.audit == null'
```

Remove `ignore_missing` and check if field `!= null`. Run pipeline tests and push expected log file.
```suggestion
    if: 'ctx.json?.enforcementSource != null'
```